scripts/package

```diff
@@ -28,6 +28,10 @@ TAG="v${APIVERSION}_`date -u +%Y%m%d`"
 REPO=${REPO:-longhornio}
 IMAGE=${REPO}/${PROJECT}:${TAG}
 
+# update base image to get latest changes
+BASE_IMAGE=`grep FROM package/Dockerfile | grep -vi AS | awk '{print $2}'`
```

Comment:
Does this only refer to the first image, now that run doesn't have the AS clause anymore?